### PR TITLE
impr(quotes): add French quotes (16ur)

### PR DESCRIPTION
**Pull request by 16ur**
Add 7 french quotes !

### CODE REVIEW

**Comment by Miodec** [comment]
Please include translations so the content can be verified

File: frontend/static/quotes/french.json [hunk 1]
```diff
@@ -1146,6 +1146,52 @@
       "source": "Orelsan - Jour meilleur",
       "length": 96,
       "id": 193
+    },
+    {
+      "text":"Je n’ai jamais rien volé... sauf une écuelle, un jour, à un chien... qui en avait deux",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1151]: "I have never stolen anything... except a bowl, one day, from a dog... who had two"

File: frontend/static/quotes/french.json [hunk 2]
```diff
@@ -1146,6 +1146,52 @@
...
+      "source":"Joseph Bouvier dans Le juge et l'assassin (1976)",
+      "length": 86,
+      "id": 194
+    },
+    {
+      "text": "On ne peut pas faire l’amour du matin au soir. C’est pour ça qu’on a inventé le travail.",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1157]: "We can't make love from morning to night. This is why we invented work."

File: frontend/static/quotes/french.json [hunk 3]
```diff
@@ -1146,6 +1146,52 @@
...
+      "source": "Dr Bicard dans 'L’Homme qui aimait les femmes' de François Truffaut (1977)",
+      "length": 88,
+      "id": 195
+    },
+    {
+      "text": "Le temps qui passe est un traître, mais il laisse des souvenirs.",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1163]: "The passage of time is treacherous, but it leaves memories."

File: frontend/static/quotes/french.json [hunk 4]
```diff
@@ -1146,6 +1146,52 @@
...
+      "source": "La Haine de Mathieu Kassovitz (1995)",
+      "length": 64,
+      "id": 196
+    },
+    {
+      "text": "Il n'y a rien de plus fort au monde que la douceur.",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1169]: "There is nothing stronger in the world than gentleness."

File: frontend/static/quotes/french.json [hunk 5]
```diff
@@ -1146,6 +1146,52 @@
...
+      "source": "Cyrano de Bergerac de Jean-Paul Rappeneau (1990)",
+      "length": 51,
+      "id": 197
+    },
+    {
+      "text": "La vie n'est qu'un long calvaire. C'est pourquoi il est essentiel d'y ajouter des moments de joie et d'amour.",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1175]: "Life is just one long ordeal. This is why it is essential to add moments of joy and love."

File: frontend/static/quotes/french.json [hunk 6]
```diff
@@ -1146,6 +1146,52 @@
...
+      "source": "Amélie Poulain de Jean-Pierre Jeunet (2001)",
+      "length": 109,
+      "id": 198
+    },
+    {
+      "text": "L'amour n'est pas seulement un sentiment, c'est un art aussi.",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1181]: "Love is not just a feeling, it is an art too."

File: frontend/static/quotes/french.json [hunk 7]
```diff
@@ -1146,6 +1146,52 @@
...
+      "source": "Anna Karénine de Léon Tolstoï",
+      "length": 61,
+      "id": 199
+    },{
+      "text": "Le bonheur n'est réel que lorsqu'il est partagé.",
```
Comment (16ur) [inline frontend/static/quotes/french.json:1186]: "Happiness is only real when shared."

**Review by 16ur — COMMENTED** [review]
I just added the quotes translation. I'm kinda new with PR so, let me know if I need to do something more :D

File: frontend/static/quotes/french.json [hunk 8]
```diff
@@ -1146,6 +1146,47 @@
...
+      "source": "Into the Wild de Sean Penn (2007)",
+      "length": 48,
+      "id": 200
     }
   ]
 }
```
Comment (NipulM) [inline frontend/static/quotes/french.json:1192]: Hello! 👋🏼

While this approach gets the job done, consider attaching the translations in the _conversation_ when submitting the pull request (PR). If you forget to provide the translations initially, you can always edit them later.

You can display the translations using the following format:

Translation of the first quote:
- "Je n’ai jamais rien volé... sauf une écuelle, un jour, à un chien... qui en avait deux"
- "I have never stolen anything... except a bowl, one day, from a dog... who had two"

Translation of the second quote:
- "On ne peut pas faire l’amour du matin au soir. C’est pour ça qu’on a inventé le travail."
- "We can't make love from morning to night. This is why we invented work."
etc.

This not only makes it easier for you but also more convenient for Miodec to review! :)

**Pull request closed without merging**